lib/compat/wordpress-6.4/block-hooks.php

```diff
@@ -302,6 +302,20 @@ function gutenberg_register_block_hooks_rest_field() {
 	);
 }
 
+/**
```

Comment:
This code should not live here. Create a new file in the `lib/compat/wordpress-6.5` directory. Maybe blocks.php.